topic/timeseries/time-series-decomposition.ipynb

```diff
@@ -106,12 +106,49 @@
     "engine = sa.create_engine(CONNECTION_STRING, echo=os.environ.get('DEBUG'))"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "First, import data into CrateDB."
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "with engine.connect() as conn:\n",
+    "    conn.execute(sa.text(\n",
+    "        \"\"\"\n",
+    "        CREATE TABLE IF NOT EXISTS weather_data (\n",
+    "            \"timestamp\" TIMESTAMP,\n",
+    "            \"location\" VARCHAR,\n",
+    "            \"temperature\" DOUBLE,\n",
+    "            \"humidity\" DOUBLE,\n",
+    "            \"wind_speed\" DOUBLE\n",
+    "        )\n",
+    "        \"\"\"\n",
+    "    ))\n",
+    "    result = conn.execute(sa.text(\"\"\"\n",
+    "        COPY weather_data\n",
+    "        FROM 'https://github.com/crate/cratedb-datasets/raw/main/cloud-tutorials/data_weather.csv.gz'\n",
+    "        WITH (format='csv', compression='gzip', empty_string_as_null=true)\n",
+    "        \"\"\"))\n",
+    "    conn.execute(sa.text(\"REFRESH TABLE weather_data\"))"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
```

Comment:
Well, this is getting boring very quickly. We had long planned to come up with a solution, and we should finally provide it.

- https://github.com/crate-workbench/cratedb-toolkit/issues/89
- https://github.com/crate/cratedb-datasets